### PR TITLE
Feature/add modal

### PR DESCRIPTION
**Pull request by Natalia504**
Created a modal component.

Desktop:
image

Mobile:
image

### CODE REVIEW

File: client/src/components/Modal/modal.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <div className='modal-body'>{modalMessage}</div>
+        <div className='modal-buttons'>
+          <button onClick={onYes} className='yes-button'>YES</button>
+          <button onClick={closeModal} className='no-button'>NO</button>
+        </div>
+      </div>
+    </div>
+  )}
+
+export default Modal;
```
Comment (Rajdeepc) [inline client/src/components/Modal/modal.jsx:17]: Add a newline at the end of this line

**Review by Rajdeepc — COMMENTED** [review]
Please attach the UI of the modal in the description

**Comment by Natalia504** [comment]
> Please attach the UI of the modal in the description

Do you mean the screenshot? Or what kind of description are you looking for?

**Review by Rajdeepc — APPROVED** [review]

**Comment by Rajdeepc** [comment]
Thank you for the contribution. Happy coding.

**Pull request merged**